src/uikit.cs

```diff
@@ -18235,6 +18330,52 @@ interface UIDocumentBrowserAction {
 		[Export ("supportsMultipleItems")]
 		bool SupportsMultipleItems { get; set; }
 	}
+
+
+	interface IUIFocusItemContainer {}
+	[iOS (12,0), TV (12,0), NoWatch]
+	[Protocol]
+	interface UIFocusItemContainer
+	{
+		[Abstract]
+		[Export ("coordinateSpace")]
+		IUICoordinateSpace CoordinateSpace { get; }
+
+		[Abstract]
+		[Export ("focusItemsInRect:")]
+		IUIFocusItem[] GetFocusItems (CGRect rect);
+	}
+
+	[iOS (12,0), TV(12,0), NoWatch]
+	[Protocol]
+	interface UIFocusItemScrollableContainer : UIFocusItemContainer
+	{
+		[Abstract]
+		[TV(9,0), NoWatch]
```

Comment:
The protocol itself is `TV (12,0)`, but there are members with `TV (9,0)`? That sounds strange (but otoh it wouldn't surprise me either...)